packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseOpenAPIObject-test.js

```diff
@@ -47,6 +47,41 @@ describe('#parseOpenAPIObject', () => {
     expect(parseResult.api.get(0).href.toValue()).to.equal('/');
   });
 
+  it('can parse a document with servers array', () => {
+    const object = new namespace.elements.Object({
+      openapi: '3.0.0',
+      info: {
+        title: 'My API',
+        version: '1.0.0',
+      },
+      paths: {},
+      servers: [
+        {
+          url: 'https://user.server.com/1.0',
+        },
+        {
+          url: 'https://user.server.com/2.0',
+          description: 'The production API server',
+        },
+      ],
+    });
+
+    const parseResult = parse(context, object);
+
+    expect(parseResult.length).to.equal(1);
+    expect(parseResult.api.title.toValue()).to.equal('My API');
+    expect(parseResult.api.length).to.equal(1);
+
+    const serversArray = parseResult.api.get(0);
```

Comment:
```suggestion
    const hostsCategory = parseResult.api.get(0);
```

As we're dealing in API Elements in these checks, I would use the API Elements terminology instead of "servers" the OAS terminology in this and below checks.

Reply:
ok changed;
it is easy to confuse the terminology while writing parsing methods.